### PR TITLE
Add build tests for melodic-devel

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
**Problem Description**
This PR adds a build test CI for the `melodic-devel` branch, since the existing CI is not able to test this branch.

**Additional Context**
- Support for ROS melodic in this branch has been added through the branch `melodic-devel`

### CODE REVIEW

**Comment by Jaeyoung-Lim** [comment]
@maximilianwulf Would it be possible to enable github actions for this repo?

File: .github/workflows/build_test.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  push:
+    branches:
+    - 'melodic-devel'
+  pull_request:
+    branches:
+    - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (maximilianwulf) [inline .github/workflows/build_test.yml:12]: Ubuntu 18.04?

**Comment by maximilianwulf** [comment]
> @maximilianwulf Would it be possible to enable github actions for this repo?

Let me check.

File: .github/workflows/build_test.yml (hunk 1 above)
Comment (Jaeyoung-Lim) [reply .github/workflows/build_test.yml:12]: That is just the environment the actions will run. I am running a container for nelodic on the next line

**Comment by Jaeyoung-Lim** [comment]
@maximilianwulf Any updates regarding this issue? Thanks in advance!

**Comment by Jaeyoung-Lim** [comment]
@maximilianwulf Any updates?

**Comment by maximilianwulf** [comment]
Yes, sorry for super long delay.

**Pull request merged**